File: test/fixtures/javascript-xhr/get_with_all_numerical_data.js
const data = '18233982904';

let xhr = new XMLHttpRequest();
xhr.withCredentials = true;
xhr.open('POST', 'http://localhost:28139/CurlToNode');
xhr.setRequestHeader('Content-Type', 'application/json');
xhr.setRequestHeader('Accept', 'application/json');

xhr.onload = function() {
  console.log(xhr.response);
};

xhr.send(data);

File: test/fixtures/r-httr2/get_with_all_numerical_data.r
library(httr2)

request("http://localhost:28139/CurlToNode") |>
  req_method("POST") |>
  req_headers(Accept = "application/json") |>
  req_body_raw(
    "18233982904",
    type = "application/json"
  ) |>
  req_perform()
